waspc/data/Generator/templates/sdk/client/webSocket/WebSocketProvider.tsx

```diff
@@ -31,7 +36,7 @@ apiEventsEmitter.on('sessionId.set', refreshAuthToken)
 apiEventsEmitter.on('sessionId.clear', refreshAuthToken)
 
 // PRIVATE API
-export const WebSocketContext = createContext({
+export const WebSocketContext: Context<WebSocketContextValue> = createContext({
```

Comment:
Interesting that it couldn't infer this type 🤨

Reply:
It could infer the type, but couldn't construct a valid declaration file without the type's name in scope:
```
Exported variable 'WebSocketContext' has or is using name 'ClientToServerEvents' 
from external module ".../todo-typescript/.wasp/out/sdk/wasp/ext-src/websocket/index" 
but cannot be named.
```
 See [this comment](https://github.com/wasp-lang/wasp/pull/1773/files#r1492398790) for more details.

Also, this export should be explicitly typed anyway according to the [module boundaries rule](https://github.com/wasp-lang/vscode-wasp/pull/6#discussion_r970774084).

I've created an issue for explicitly typing our public API in SDK: https://github.com/wasp-lang/wasp/issues/1776